inspirehep/config.py

```diff
@@ -1136,6 +1136,11 @@
     'array': (list, tuple),
 }
 
+# If set, will disable downloading records files by default (can be overriden
+# by the `skip_files` parameter of `InspireRecord.create` and
+# `InspireRecord.update`)
```

Comment:
A docstring after the configuration variable will show up when building the docs (e.g. http://inspirehep.readthedocs.io/en/latest/_api/inspirehep.modules.hal.html#module-inspirehep.modules.hal.config), while a comment will be ignored. So I'd say it's better to document variables with docstrings.